account.go

```diff
@@ -33,6 +33,10 @@ func (a *Account) Address() common.Address {
 	return crypto.PubkeyToAddress(a.pk.PublicKey)
 }
 
+func (a *Account) PrivKey() *ecdsa.PrivateKey{
```

Comment:
Can you run gofmt on this file please? Will handle the missing space before the end bracket 😄

Reply:
Done ;)